static/dbstructure.config.php

```diff
@@ -609,6 +609,39 @@
 			"done_pid_priority_created" => ["done", "pid", "priority", "created"]
 		]
 	],
+	"dcontact" => [
+		"comment" => "Diaspora compatible contacts - used in the Diaspora implementation",
+		"fields" => [
+			"uri-id" => ["type" => "int unsigned", "foreign" => ["item-uri" => "id"], "comment" => "Id of the item-uri table entry that contains the dcontact url"],
```

Comment:
I guess you have to set `"not null" => "1", "primary" => "1",`as well, since this is the primary key and because of this it mustn't be `null`.

I guess I had been really bad in communicating over the last few days. When I mentioned renaming, I meant especially the table, since this will hopefully be a part of the upcoming `account` mechanism. The next step would be a replacement for `apcontact` and then a very basic `account` table with only very few fields since all protocol relevant fields will be stored in the protocol account side tables. This should save some space.

Reply:
Thank you for the elaboration, but I'm not sure what needs to be done beyond the precise changes you've outlined in your first paragraph.

Reply:
What I mean is: The view is called "diaspora-contact-view", the table is called "dcontact". It would be better if "dcontact" was renamed to "diaspora-contact".